src/da65.vcxproj

```diff
@@ -109,6 +110,7 @@
     <ClInclude Include="da65\handler.h" />
     <ClInclude Include="da65\infofile.h" />
     <ClInclude Include="da65\labels.h" />
+    <ClInclude Include="da65\opc4510.h" />
```

Comment:
I guess this change isn't actually related to this pull request but rather a fix/addon to a 3rd party change. I generally prefer a separate pull request in such a case but I'm willing to merge this here.